src/ngRoute/route.js

```diff
@@ -160,7 +160,7 @@ function $RouteProvider() {
    */
   this.when = function(path, route) {
     //copy original route object to preserve params inherited from proto chain
-    var routeCopy = angular.copy(route);
+    var routeCopy = route;
```

Comment:
I think the intention of using `copy` instead of `extend` was to preserve inherited properties (talk about using a sledgehammer to crack a nut :smiley:). `shallowCopy` (which we need to copy over to `angular-route` btw) would also shallow-copy inherited properties, so we are covered.